### PR TITLE
fix: golangci-lint error

### PR DESCRIPTION
**Pull request by FrankYang0529**
#### Which issue(s) this PR fixes:
https://github.com/longhorn/longhorn/issues/7448

### CODE REVIEW

File: sparse/test/client_test.go [hunk 1]
```diff
@@ -2,12 +2,13 @@ package test
...
 import (
 	"context"
-	"math/rand"
+	"crypto/rand"
 	"os"
 	"testing"
 
 	. "github.com/longhorn/sparse-tools/sparse"
 	"github.com/longhorn/sparse-tools/sparse/rest"
+	"github.com/stretchr/testify/assert"
```
Comment (derekbit) [inline sparse/test/client_test.go]:
```suggestion
	"crypto/rand"
	"os"
	"testing"

	"github.com/stretchr/testify/assert"

	. "github.com/longhorn/sparse-tools/sparse"
	"github.com/longhorn/sparse-tools/sparse/rest"
```

Comment (FrankYang0529) [reply sparse/test/client_test.go]: Fixed it. Thank you.

File: Dockerfile.dapper [hunk 2]
```diff
@@ -11,16 +11,16 @@ RUN apt-get update && \
...
 
 RUN wget -O - ${!DOCKER_URL} > /usr/bin/docker && chmod +x /usr/bin/docker
 
-ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm=armv6l GOLANG_ARCH=GOLANG_ARCH_${ARCH} \
+ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm64=arm64 GOLANG_ARCH=GOLANG_ARCH_${ARCH} \
     GOPATH=/go PATH=/go/bin:/usr/local/go/bin:${PATH} SHELL=/bin/bash
 
-RUN wget -O - https://storage.googleapis.com/golang/go1.17.10.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local && \
-    go get github.com/rancher/trash && GO111MODULE=on go get golang.org/x/lint/golint@v0.0.0-20210508222113-6edffad5e616
+RUN wget -O - https://storage.googleapis.com/golang/go1.22.2.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local
```
Comment (derekbit) [inline Dockerfile.dapper:22]: Updating the version in each component manually is quite tedious.
Is it possible that we can define the version in a single definition file of one repo? This way, Dockerfile.dapper in all components could refer directly to this file, simplifying the process.

cc @innobead

**Review by derekbit — APPROVED** [review]
LGTM

File: Dockerfile.dapper (hunk 2 above)
Comment (FrankYang0529) [reply Dockerfile.dapper:22]: Also golanci-lint version.

Comment (derekbit) [reply Dockerfile.dapper:22]: Can you help create a ticket for the improvement?
This PR LGTM.

**Pull request merged**